README.md

```diff
@@ -1,2 +1,15 @@
-# how-to-create-react-accumulation-charts-of-syncfusion
-A quick-start react project that depicts the creation of Syncfusion Accumulation Charts with appropriate title, legend, data labels, and tooltip.
+# How to Create React Accumulation Charts of Syncfusion
+
+A quick-start project that allows you to create an accumulation charts of Syncfusion in the react application. This project also contains a code snippet to add a title, legend, data label, and tooltip to the pie chart.
```

Comment:
Revisit the sentence, as I can notice grammatical errors. Also, at last you just mentioned pie chart alone.

Reply:
Done the changes